applications/directory/src/pages/profile/[token]/edit.js

```diff
@@ -51,19 +57,51 @@ const ProfileReviewEditPage = () => {
             <h1>Edit artist profile</h1>
           </Box>
 
-          <Box
-            sx={{
-              display: 'flex',
-              justifyContent: 'space-between',
-              alignItems: 'flex-start'
-            }}
-          >
-            <CreateProfileForm />
-          </Box>
+          <Card elevation={6}>
+            <Loader
+              isLoading={
```

Comment:
Let's create and use `loading` and error variable:

```javascript
const loading = categoriesLoading || tagsLoading || skillsLoading || artistLoading;
const error = categoriesError || tagsError || skillsError || artistError;
```

Reply:
done